test/creatingEntityClass.spec.js

```diff
@@ -39,4 +39,21 @@ describe('creating an entity class', () => {
       expect(User.staticProperty).to.equal('I am a static property');
     });
   });
+
+  describe('using default values for attributes', () => {
```

Comment:
What do you think about adding two contexts here? One for when it's using a normal value and another when it's using a function. That way it will be easier to contributors/users to see all the features separately.